### PR TITLE
Enable Stackgres Mainnet NA 

### PR DESCRIPTION
**Pull request by jnels124**
* enable stackgres operator

### CODE REVIEW

**Review by jascks — APPROVED** [review]
LGTM

File: clusters/mainnet-na/common/helmrelease.yaml [hunk 1]
```diff
@@ -61,6 +61,8 @@ spec:
                   regex: (api_.+_bucket|executor.*|lettuce_.*_bucket|traefik_.+_bucket)
                   sourceLabels: [ __name__ ]
           replicaExternalLabelName: __replica__
+    stackgres:
```
Comment (xin-hedera) [inline clusters/mainnet-na/common/helmrelease.yaml:64]: should we also increate prometheus memory limits?

Comment (jnels124) [reply clusters/mainnet-na/common/helmrelease.yaml:67]: yes probably a good time to do that as well. Done

**Review by jascks — APPROVED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**